builder/common/block_device.go

```diff
@@ -33,21 +33,25 @@ const (
 // HCL2 example:
 //
 // ```hcl
-// launch_block_device_mappings {
-//     device_name = "/dev/sda1"
-//     encrypted = true
-//     kms_key_id = "1a2b3c4d-5e6f-1a2b-3c4d-5e6f1a2b3c4d"
-// }
+//
+//	launch_block_device_mappings {
+//	    device_name = "/dev/sda1"
+//	    encrypted = true
+//	    kms_key_id = "1a2b3c4d-5e6f-1a2b-3c4d-5e6f1a2b3c4d"
+//	}
+//
```

Comment:
While I generally am not against using a linter for cleaning-up files, in this case, updating the documentation feels out of the scope of this PR, and makes the diff harder to read.
Could you please leave the doc changes out of this PR? If you think they are relevant, maybe open a separate PR for us to review, so this doesn't become a blocker for this one?